crates/turborepo-repository/src/package_graph/mod.rs

```diff
@@ -30,6 +30,21 @@ pub struct PackageGraph {
     lockfile: Option<Box<dyn Lockfile>>,
 }
 
+#[derive(Eq, PartialEq, Hash)]
+pub struct WorkspacePackage {
```

Comment:
I'm not in love with this name because it implies that this is a workspace's package, like how `WorkspaceName` implies it's a workspace's name. When really it's a workspace. Maybe we can just call it `Workspace`?

Reply:
I should add a comment. Basically all the _other_ names are wrong, and "workspace" should be replaced with "package" everywhere. I tried to just use `Package`, but I think it was conflicting with `turborepo_lockfiles::Package` and I didn't want to figure it out.

Would you be ok with a comment above this and some of the other structs? 

https://vercel.com/docs/vercel-platform/glossary#workspace
https://vercel.com/docs/vercel-platform/glossary#package

That said... this is _still_ not the best name because "root" is not a workspace package either.

Reply:
Hmm, I feel like this rename should be done all at once though. Because otherwise it'll be confusing